action.yml

```
@@ -167,6 +167,22 @@ runs:
         sudo ln -svf caproto-repeater "/bin/caRepeater"
         echo "::endgroup::"
 
+    - name: Apply auto settings
+      shell: bash -leo pipefail {0}
+      run: |
+        echo "::group::Applying auto settings"
+        if [ -f "${CLONNED_REPO}/.ci/auto_settings.sav" ]; then
```

We fixed the spelling of the variable in #11:
```suggestion
        if [ -f "${CLONED_REPO}/.ci/auto_settings.sav" ]; then
```